web/packages/shared/components/FieldMultiInput/FieldMultiInput.tsx

```diff
@@ -21,15 +21,35 @@ import { ButtonSecondary } from 'design/Button';
 import ButtonIcon from 'design/ButtonIcon';
 import Flex from 'design/Flex';
 import * as Icon from 'design/Icon';
-import Input from 'design/Input';
 import { useRef } from 'react';
 import styled, { useTheme } from 'styled-components';
 
+import {
+  precomputed,
+  Rule,
+  ValidationResult,
+} from 'shared/components/Validation/rules';
+import { useRule } from 'shared/components/Validation';
+
+import FieldInput from '../FieldInput';
+
+type StringListValidationResult = ValidationResult & {
+  /**
+   * A list of validation results, one per label. Note: items are optional just
+   * because `useRule` by default returns only `ValidationResult`. For the
+   * actual validation, it's not optional; if it's undefined, or there are
+   * fewer items in this list than the labels, the corresponding items will be
+   * treated as valid.
+   */
+  items?: ValidationResult[];
```

Comment:
I think if i received this field in some result i would have no idea what it meant. Based on the comment, should this be `validItems` or ... unchecked items? (based on label number? im confused)

Reply:
@avatus How about `itemResults`?

Reply:
what is the context of the result? valid or not valid? checked vs unchecked? thats the part I'm getting lost with (forgive my brain)

Reply:
Oh, I see what you mean. The results are "valid or not valid". In other words, these are per-item validation results that enable us to show a validity state and message for each item of the string list separately.

Reply:
Ahhhh, that makes sense. with that context, `items` probably makes most sense. I think maybe even `results` might be better but, I'll leave it up to you if you want to make any changes